src/applications/appeals/995/containers/App.jsx

```diff
@@ -133,17 +135,22 @@ export const App = ({
 
   let content = (
     <RoutedSavableApp formConfig={formConfig} currentLocation={location}>
-      <ITFWrapper
-        loggedIn={loggedIn}
-        pathname={pathname}
-        title={formConfig.title}
-        benefitType={subTaskBenefitType}
-        router={router}
-        accountUuid={accountUuid}
-        inProgressFormId={inProgressFormId}
+      <DowntimeNotification
+        appTitle="Supplemental Claims"
+        dependencies={[externalServices.decision_reviews]}
```

Comment:
This should be:
```suggestion
        dependencies={[externalServices.decisionReviews]}
```

And you'll need to add the following to `src/platform/monitoring/DowntimeNotification/config/externalServices.js`
```
decisionReviews: 'decision_reviews',
```

In order for this to work properly.

Reply:
Updated!